File: src/client_gen/java/MyStrategy.java
import model.*;

public class MyStrategy {
    public Action getAction(PlayerView playerView, DebugInterface debugInterface) {
        throw new UnsupportedOperationException("Not implemented");
    }
    public void debugUpdate(PlayerView playerView, DebugInterface debugInterface) {
        debugInterface.send(new DebugCommand.Clear());
        debugInterface.getState();
    }
}

File: src/client_gen/scala/MyStrategy.scala
class MyStrategy {
  def getAction(playerView: model.PlayerView, debugInterface: Option[DebugInterface]): model.Action = ???
  def debugUpdate(playerView: model.PlayerView, debugInterface: DebugInterface) {
    debugInterface.send(model.DebugCommand.Clear())
    debugInterface.getState()
  }
}